src/api/app/models/package.rb

```diff
@@ -184,8 +184,20 @@ def self.internal_get_project(project)
   # You can follow this type of project link and try to find the Package from the "maintenance update"
   # Project by setting in the opts hash:
   #   check_update_project: true
+  #
+  # It will ignore "scmsync" Projects and not "follow" this type of project link to find the Package.
+  # https://github.com/openSUSE/open-build-service/wiki/Links#project-scm-bridge-links
+  #
+  # You can follow this type of project link and try to find the Package from the "SCM" Project
+  # by setting in the opts hash:
+  #   follow_project_scmsync_links: true
```

Comment:
please name it more neutral like "use_backend_only_packages: true" as this is also the case for any package from remote instances. So we should not be scmsync specific here.

Reply:
Your was, this is not. This is limited to scmsync.